Ansible/init_docker/tasks/main.yml

```diff
@@ -107,22 +109,28 @@
     msg: "Nginx is already installed"
   when: check.changed == False
 
-- name: Copy NGx config file defaults
-  ansible.builtin.copy:
-    src: "{{ item }}"
-    dest: /etc/nginx/sites-available  # link to /path/to/file
-    follow: true
-  loop:
-    - "{{ playbook_dir }}/../MyWeb/default"
+- name: Create Nginx SSL configuration
+  ansible.builtin.template:
+    src: nginx_ssl.conf.j2
+    dest: "{{ nginx_config_file }}"
+
+- name: Enable the Nginx site configuration
+  ansible.builtin.command: ln -s "{{ nginx_config_file }}" "{{ nginx_enabled_config }}"
+  args:
+    creates: "{{ nginx_enabled_config }}"
+  notify: Restart Nginx
+
+- name: Obtain SSL certificate for domain with Certbot
+  ansible.builtin.command: >
+    certbot --nginx -d {{ item }} --email {{ email }} --agree-tos --non-interactive
+  loop: "{{ domain_name}}"
```

Comment:
I fixed the problem. Check it pls.

Reply:
why not use 

`{{ domain_name | join(',') }}`
instead of 
`-d {{ domain }} -d www.{{ domain }}`


Take into account a scenario where you want to add an additional sub-domain, such as todo.webma.sk.

Reply:
also consider using a better name for the variable `domain_name`, e.g. `domains`...

Reply:
i fixed the problem...aae25f74b5975ad5750bdaf24ee535c86e5b453f